background/about-boilerplates.rst

```diff
@@ -0,0 +1,151 @@
+About Boilerplates
+==================
+
+As well as Django and Python applications, our Docker containerisation can
+include and build frontend components into your projects.
+
+Your project's *Boilerplate* will define the components - HTML, CSS and
+JavaScript - that are set up each time the project is built.
+
+Every Divio Cloud project includes a Boilerplate, whether it's one of our
```

Comment:
Maybe mention something along the line "Even if its just a blank boilerplate with the simplest of configuration"?